cmd/imagestream.go

```diff
@@ -33,12 +39,15 @@ func NewImageStreamCleanupCommand() *cobra.Command {
 		Run:     o.cleanupImageStreamTags,
 	}
 	cmd.Flags().BoolVarP(&o.Force, "force", "f", false, "delete image stream tags")
-	cmd.Flags().IntVarP(&o.CommitLimit, "git-commit-limit", "l", 100, "only look at the first <n> commits to compare with tags or use -1 for all commits")
+	cmd.Flags().IntVarP(&o.CommitLimit, "git-commit-limit", "l", 0, "only look at the first <n> commits to compare with tags or use 0 for all commits")
 	cmd.Flags().StringVarP(&o.RepoPath, "git-repo-path", "p", ".", "absolute path to Git repository (for current dir use: $PWD)")
 	cmd.Flags().StringVarP(&o.Namespace, "namespace", "n", "", "Kubernetes namespace")
 	cmd.Flags().IntVarP(&o.Keep, "keep", "k", 10, "keep most current <n> images")
 	cmd.Flags().BoolVarP(&o.Tag, "tag", "t", false, "use tags instead of commit hashes")
 	cmd.Flags().StringVar(&o.Sorted, "sort", string(git.SortOptionVersion), "sort tags by criteria. Allowed values: [version, alphabetical]")
+	cmd.Flags().BoolVarP(&o.Orphan, "orphan", "o", false, "delete images that does not match any git commit")
+	cmd.Flags().StringVar(&o.Duration, "older-than", "", "delete images that are older than the duration. Ex.: [1y2mo3w4d5h6m7s]")
+	cmd.Flags().StringVarP(&o.Regex, "include", "i", "^[a-z0-9]{40}$", "delete images that match the regex, works only with the -o flag, defaults to matching Git SHA commits")
```

Comment:
The flag name `include` absolutely makes no reference as to what is "included". If it's about the "include image tags in deletion set", then I would suggest something more meaningful like `--orphan-deletion-pattern`